SparkPost/matchbox
# UX-633, UX-617, UX-618 Convert Layout, Columns and Column components to TypeScript

### What Changed

- Converted `Layout` component to TypeScript
- Converted `Columns` component to TypeScript
- Converted `Column` component to TypeScript

### How To Test or Verify

- Make sure tests pass
- Run locally `npm run start:libby` and check Layout, Columns and Column stories

### PR Checklist
